packages/components/src/hooks/UIProvider.tsx

```diff
@@ -13,14 +13,16 @@ interface State {
   cart: boolean
   /** Region modal */
   modal: boolean
+  /** Add review modal */
+  addReviewModal: boolean
```

Comment:
```suggestion
  /**Review modal */
  reviewModal: boolean
```

**todo**: rename to reviewModal